### PR TITLE
doc: Improve starting documentation

### PR DESCRIPTION
**Pull request by tahini**
* Add a CONTRIBUTING.md file with instructions to contribute to the community, code, test and debug the application
* Update the main README to contain all the documentation required to run a new instance of Transition.

### CODE REVIEW

**Comment by tahini** [comment]
@kaligrafy, @greenscientist With this, I think this completes the documentation to get started with a new instance of Transition. It's the necessary and the sufficient for someone to go from getting the code to opening the application in a browser.

For the rest, we still need a complete user documentation, which is not the scope here.

**Comment by brodyFlannigan** [comment]
Hi!

The additionnal documentation is really good and is sufficient to allow me (or any other user) to deploy Transition.

File: README.md [hunk 1]
```diff
@@ -13,45 +13,85 @@ For Ubuntu users: [complete step-by-step development environment setup procedure
...
 * yarn: [debian/ubuntu](https://classic.yarnpkg.com/en/docs/install/#debian-stable) [macos](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
-* Rust (it is used to run the json2capnp cache service which makes the application much faster if there's a lot of transit data)
+* Rust: It is used to run the json2capnp cache service which makes the application much faster if there's a lot of transit data.
 
 ## Installation
 
-* Install Node and Yarn
+* Install dependencies:
+```
+sudo apt-get install postgresql postgis lua5.3 liblua5.3-dev \
```
Comment (greenscientist) [inline README.md:25]: This is ubuntu specific, it should be mentionned. (Something like an example for Ubuntu xx.xx)
This already does not work for Debian/bullseye, as lua5.4 is used.

File: README.md [hunk 2]
```diff
@@ -77,7 +117,7 @@ Copy a geojson polygon
 #TODO Maybe the whole projects directory should be in a volume
 docker cp /path/to/my/random_polygon.geojson transition_transition-www_1:/app/projects/demo_transition/imports/polygon.geojson
 ### TODO ADD  downloadosmtask....
```
Comment (greenscientist) [inline README.md:119]: We could take the opportunity to replace this TODO line with the actual download task.

**Review by greenscientist — COMMENTED** [review]
Just a couple of small comments

**Review by greenscientist — APPROVED** [review]

**Pull request merged**